python/hsfs/constructor/fs_query.py

```diff
@@ -16,7 +16,7 @@
 
 import humps
 from hsfs import engine
-from hsfs.constructor import hudi_feature_group_alias, on_demand_feature_group_alias
+from hsfs.constructor import hudi_feature_group_alias, external_group_alias
```

Comment:
```suggestion
from hsfs.constructor import hudi_feature_group_alias, external_feature_group_alias
```
is it missing a `feature` ?